### DISCUSSION TITLE
[ENG-346] Modal Integration

### DISCUSSION THREAD

**Discussion by mdavish** [Infisical/infisical#5629] [Feature Request]
### Feature description

It would be great if you offered an integration with [Modal](https://modal.com/). Modal offers the ability to define "secrets" and each secret may contain many environment variables.

### Why would it be useful?

Modal is a popular new platform for running Python code in the cloud. They have become quite popular for AI applications and we've been using them to run various web servers and background jobs. Obviously all this code needs access to our secrets which we keep in Infisical, but right now you have to do a fair amount of hacking in order to inject Infisical secrets into Modal.

<sub>From [SyncLinear.com](https://synclinear.com) | [ENG-346](https://linear.app/infisical/issue/ENG-346/modal-integration)</sub>

**Comment by kisharnath** [comment]
@vmatsiiako  can I work on this?

**Comment by Salman2301** [comment]
I could be wrong, I don't think we can update the environment / secret through rest API for Modal atm. Available options are cli or python? but that needs to run in local server.

**Comment by abendrothj** [comment]
If this is still up for grabs, I'll start working on it!

[a comment by an automated account is not shown]
